### DISCUSSION TITLE
wpf webview2使用表单编辑弹框报错

### DISCUSSION THREAD

**Discussion by 925934787** [dotnetcore/BootstrapBlazor#1489] [Q&A] [answered]
使用脚手架生产的项目，新建了一个wpf client，和server公用一个shared，wpf这边表单打开编辑弹框报错
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/19659397/5764356d-26e1-4c3b-a770-2876ce2aac19)
使用ServerSide打开可以正常显示

**Comment by Vision-Zhang** [comment]
@925934787 App.razor   加入 BootstrapBlazorRoot 标签

**Reply by 925934787** [reply]
wpf 是用的shared blazor类库的app.razor ,我是需要在wpf client下面也要增加一个App.razor 吗？
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/19659397/8e619b8b-4706-4699-9435-743be8bf21a6)

**Comment by Vision-Zhang** [comment]
@925934787 不需要 传一个示例项目上来把

**Reply by 925934787** [reply]
[BootstrapBlazorApp1.zip](https://github.com/dotnetcore/BootstrapBlazor/files/11943975/BootstrapBlazorApp1.zip)

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/19659397/384d968d-c4ab-487d-901e-281b3930577f)

**Comment by Vision-Zhang** [comment]
@925934787  https://github.com/densen2014/BlazorMaui  参考一下这个例子

**Reply by 925934787** [reply, marked as answer]
谢谢，找到问题了了，确实是和BootstrapBlazorRoot有关系，wpf那边我绑定错了
